db/migrate/20230901195553_add_encrypted_user_attributes_to_oauth_sessions.rb

```diff
@@ -0,0 +1,5 @@
+class AddEncryptedUserAttributesToOAuthSessions < ActiveRecord::Migration[6.1]
+  def change
+    add_column :oauth_sessions, :encrypted_user_attributes, :string
```

Comment:
We use the [lockbox gem](https://github.com/ankane/lockbox) so this column will need to be type `:text` and the name needs to end in `_ciphertext`. We can just call it `user_attributes_ciphertext`

Reply:
Here's an example: https://github.com/department-of-veterans-affairs/vets-api/pull/11559

Reply:
I don't think `t.text :encrypted_kms_key` is necessary though